File: src/main/kotlin/me/fungames/jfortniteparse/fort/objects/ItemCategoryMappingData.java
package me.fungames.jfortniteparse.fort.objects;

import me.fungames.jfortniteparse.fort.enums.EFortItemType;
import me.fungames.jfortniteparse.ue4.assets.UStruct;
import me.fungames.jfortniteparse.ue4.objects.core.i18n.FText;

@UStruct
public class ItemCategoryMappingData {
    public EFortItemType CategoryType;
    public FText CategoryName;
}

File: src/main/kotlin/me/fungames/jfortniteparse/fort/objects/ItemCategoryMappingData.kt
package me.fungames.jfortniteparse.fort.objects

import me.fungames.jfortniteparse.ue4.assets.UStruct
import me.fungames.jfortniteparse.ue4.objects.core.i18n.FText
import me.fungames.jfortniteparse.ue4.objects.uobject.FName

@ExperimentalUnsignedTypes
@UStruct
class ItemCategoryMappingData {
    var CategoryType: FName? = null // EFortItemType
    var CategoryName: FText? = null
}